ivy/functional/backends/tensorflow/activations.py

```diff
@@ -74,3 +74,20 @@ def deserialize(
     name: Union[str, None], /, *, custom_objects=Union[ivy.Dict, None]
 ) -> Union[ivy.Callable, None]:
     return tf.keras.activations.deserialize(name, custom_objects)
+
+
+def get(identifier: Union[str, ivy.Callable, None], /, *,
+        custom_objects=Union[ivy.Dict, None]
+) -> Union[ivy.Callable, None]:
```

Comment:
When splitting the function arguments over several lines, all arguments should be on a separate line. See the deserialize function in the same file for example.